packages/widgetbook/lib/src/fields/list_field.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:collection/collection.dart';
+import 'package:flutter/widgets.dart';
+import 'package:widgetbook_core/widgetbook_core.dart';
+
+import 'field.dart';
+import 'field_codec.dart';
+import 'field_type.dart';
+
+typedef LabelBuilder<T> = String Function(T value);
+
+class ListField<T> extends Field<T> {
+  ListField({
+    required super.group,
+    required super.name,
```

Comment:
What was the reasoning for the rename?

Reply:
It was hard to rename `TextField`, `NumberField`, `ColorField` if we used widget-based naming, so I went with data-type-based naming.